pagopa/pn-frontend
# localized sercq send event in timeline - pn-12539

## Short description
Localized digital progress and feedback events, when the notification is sent through SERCQ SEND channel

## List of changes proposed in this pull request
- Added SERCQ SEND case into SendDigitalDomicileStep class
- Added SERCQ SEND case into SendDigitalFeedbackStep class
- Updated tests
- Added localizations

## How to test
Login with pf/pg -> activate the sercq send channel -> login with pa and send a notification to the previous user -> check that in pa/pf/pg the notification timeline has the right copies
